workflow/snakemake_rules/trigger.smk

```diff
@@ -14,14 +14,19 @@ These output files are empty flag files to force Snakemake to run the trigger ru
 rule trigger_rebuild_pipeline:
     message: "Triggering nextstrain/ncov rebuild action (via repository dispatch)"
     input:
-        f"data/{database}/upload.done"
+        metadata_upload = f"data/{database}/metadata.tsv.gz.upload",
```

Comment:
~metadata_upload = f"data/{database}/metadata.tsv.zst.upload",~

~Let's use zst compressed upload as it's:
a) the future
b) done faster so we gain an hour or so~

Reply:
Ignore what I said above - since ncov currently uses the `gz` output we need that one to be ready!